projects/plugins/jetpack/_inc/client/recommendations/timer/index.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import React, { useState, useEffect, useCallback, useRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+const Timer = ( { className, expiryDate } ) => {
+	const d = new Date( expiryDate );
```

Comment:
Will the expiryDate value be in UTC or adjusted for local time?
If it's in UTC, we may need to factor in the local timezone offset when setting duration.

Reply:
Value's in UTC indeed. The `Date` constructor takes care of applying the timezone detected by the browser. Were you talking about the site timezone?